pkgs/by-name/jo/job-security/package.nix

```diff
@@ -0,0 +1,31 @@
+{ lib
+, stdenv
+, rustPlatform
+, fetchFromGitHub
+}:
+
+rustPlatform.buildRustPackage rec {
+  pname = "job-security";
+  version = "unstable-0-2024-03-06";
+
+  src = fetchFromGitHub {
+    owner = "yshui";
+    repo = "job-security";
+    rev = "50163cb4259b86aa3c9bd4eaccef83d73a0747db";
+    hash = "sha256-OXLL6jMCc+1XkOZjK/FQY1hBthi1CkWt/CX8uqf3C/E=";
+  };
+
+  cargoHash = "sha256-s/oL2NYZXPVQSzdH3p815156kYmLJ294edh7vBzRNT4=";
+
+  # unresolved import `ptyprocess`
```

Comment:
Why not add the python package to nativeCheckInputs?

Reply:
It doesn't appear to be related to the python package:

```
running 1 test
test server/src/pty.rs - pty::PtyProcess::spawn (line 48) - compile ... FAILED

failures:

---- server/src/pty.rs - pty::PtyProcess::spawn (line 48) stdout ----
error[E0432]: unresolved import `ptyprocess`
 --> server/src/pty.rs:50:5
  |
4 | use ptyprocess::PtyProcess;
  |     ^^^^^^^^^^ use of undeclared crate or module `ptyprocess`

error: aborting due to previous error

For more information about this error, try `rustc --explain E0432`.
Couldn't compile the test.

failures:
    server/src/pty.rs - pty::PtyProcess::spawn (line 48)

test result: FAILED. 0 passed; 1 failed; 0 ignored; 0 measured; 0 filtered out; finished in 0.08s
```

Reply:
Oh, there is a rust crate also named like that. That's indeed an upstream error.

Reply:
Fixed upstream